config/swagger.yaml

```diff
@@ -694,10 +696,11 @@ paths:
         - application/json
       parameters:
         - in: query
-          name: tx_objects
+          name: tx_encoding
           description: 'Transactions encoding: MessagePack (default) or JSON'
```

Comment:
Should this not say "message_pack" and "json", i.e. the actual values?

Reply:
It should and it does. [3 lines bellow](https://github.com/aeternity/epoch/pull/644/files#diff-89b8ec7c2f28f3e3c7d21ca9bf070bb7R491):
```
enum: [message_pack, json]
```